src/pages/SigningOfficialResearchers.js

```diff
@@ -34,7 +37,9 @@ export default function SigningOfficialResearchers() {
   return (
     div({style: Styles.PAGE}, [
       div({style: {}, className: 'signing-official-tabs'}, [
-        h(SigningOfficialTable, {researchers, signingOfficial, isLoading}, [])
+        h(SigningOfficialTable, {researchers, signingOfficial, isLoading}, []),
+        // will automatically pop up if SO has not signed
+        h(SigningOfficialDAAPopup, {})
```

Comment:
In my opinion, we should instead have acceptance be passed into this component from the parent console. Then we can very easily render either one depending on status. Same feedback for the other table components as well.

Second bit of feedback, we should probably rename `SigningOfficialDAAPopup` to something more appropriate, like `SigningOfficialAgreements`.

Reply:
Ok, I can do that. My only "ick" with that is that there is duplicate logic in three different console pages, but it does clean up the idea of the modal and such. I will go with this approach

Reply:
I figured out how to do this in a way that I think is much better than before: I wrapped the consoles in a wrapper component, which queries to see agreement status, and conditionally renders its children or the agreement information. This means that the signing official consoles require no extra logic while still being able to replace their content effectively. Let me know if you think this is the right approach